packages/orm/src/drivers/PostgresDriver.ts

```diff
@@ -227,7 +228,11 @@ export class PostgresDriver implements Driver {
           await batchInsert(knex, meta, todo.inserts);
         }
         if (todo.updates.length > 0) {
-          todo.updates.forEach((e) => (e.__orm.data["updatedAt"] = updatedAt));
+          todo.updates.forEach((e) => {
+            // Should we just go through a setter?
+            e.__orm.originalData["updatedAt"] = e.__orm.data["updatedAt"];
+            e.__orm.data["updatedAt"] = updatedAt;
```

Comment:
Previously when we bumped `updatedAt`, we weren't putting the old value in `originalData`, but we need the old value to do the "WHERE id = ... and updated_at = original_updated_at"